ui/tests/unit/lib/attach-capabilities-test.js

```diff
@@ -0,0 +1,161 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import attachCapabilities from 'vault/lib/attach-capabilities';
+import apiPath from 'vault/utils/api-path';
+import { get } from '@ember/object';
+
+let MODEL_TYPE = 'test-form-model';
+
+module('Unit | lib | attach capabilities', function(hooks) {
+  setupTest(hooks);
+
+  test('it attaches passed capabilities', function(assert) {
+    let mc = this.owner.lookup('service:store').modelFor(MODEL_TYPE);
+    mc = attachCapabilities(mc, {
+      updatePath: apiPath`update/{'id'}`,
+      deletePath: apiPath`delete/{'id'}`,
```

Comment:
Hey @meirish , sorry bit late here, I took a glance at this this morning and noticed you are missing some `$`s here. Pretty sure it doesn't affect anything here, but just if you want to fix that up at some point in the future. Oh, there's another just below also L:34-35

Ta,
John

Reply:
Sorry, just seeing this - and thanks! Ha yeah they don't matter to these tests because the tests aren't looking at the output of the template fn, but will get them updated!